.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+---
+name: CI
+"on":
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  molecule:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python_version: ["3.7"]
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+        with:
+          path: ansible_collections/middleware_automation/redhat_csp_download
+
+      - name: Set up Python ${{ matrix.python_version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python_version }}
+
+      - name: Install molecule
```

Comment:
We do not need to install `molecule` anymore. Step title should be changed and remove the pip module